### DISCUSSION TITLE
Forced Login can't be disabled?

### DISCUSSION THREAD

**Discussion by IggyTheWolf** [Arksine/moonraker#455] [Q&A] [answered]
Hello,

I just switched my printer to a new network and I am having a problem accessing it from it's .local address. I get the following screen:
<img width="594" alt="Screen Shot 2022-07-06 at 7 07 03 PM" src="https://user-images.githubusercontent.com/71224368/177675553-e8632217-5af0-4ea2-944a-a6e06a3122c9.png">

I have made the correct changes to the moonraker.conf file below:
<img width="301" alt="Screen Shot 2022-07-06 at 7 08 57 PM" src="https://user-images.githubusercontent.com/71224368/177675644-9d799ee6-8dca-4ed2-ab61-2c8368718feb.png">

I can't figure out why logins are being forced or how to turn them off? As a result I can't direct upload from SuperSlicer as well.

**Comment by matmen** [comment, marked as answer]
Hey, this is a known issue in the [v1.17.0](https://github.com/fluidd-core/FluiddPI/releases/tag/v1.17.0) FluiddPi (and later) image. You will need to add your IPv6 subnet to the trusted_clients as we're still unsure how to fix this.

**Reply by IggyTheWolf** [reply]
Thank you!

**Reply by IggyTheWolf** [reply]
@matmen I added this to end of trusted_clients and everything is working now `::/0`

**Comment by matmen** [comment]
FWIW, a temporary fix has been released in the 1.19.0 FluiddPi image
